coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/Metric.scala

```diff
@@ -20,7 +21,7 @@ case class CoordinatedMetric(collocationGroup: Set[String], coordinator: Coordin
         coordinator.secondaryMetrics(storeId, dataset).fold(throw _, identity).map { metric =>
           Cost(moves = 1, totalSizeBytes = metric.totalSizeBytes)
         }
-      }.reduceOption(Cost.max).getOrElse(Cost.Unknown) // TODO: note we way want to replace Unknown here in the future with and error
+      }.reduceOption(costOrdering.max).getOrElse(Cost.Unknown) // TODO: note we way want to replace Unknown here in the future with and error
```

Comment:
Why not have it error now?

Reply:
would rather have something that works for people trying to use it